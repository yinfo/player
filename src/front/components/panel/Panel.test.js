// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Panel from './Panel';

describe('Panel component', () => {
  it('renders as expected', () => {
    const props = {
      hasTitle: true,
      title: 'title',
    };
    const component = shallow(
      <Panel {...props}>
        <p>a child</p>
      </Panel>,
    );
    expect(component).toMatchSnapshot();
  });
});
