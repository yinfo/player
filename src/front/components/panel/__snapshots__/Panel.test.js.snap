// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel component renders as expected 1`] = `ShallowWrapper {}`;
